exercises/branching-conditionals/letterGrade/letterGrade.js

```diff
@@ -15,11 +15,46 @@
  */
 function letterGrade(percentGrade) {
   // This is your job. :)
-}
+  let gradeLetter = '';
 
+  if(percentGrade >= 97){
```

Comment:
If you check in the reverse order, it becomes cleaner and easier to modify should you ever want to change where the grade cutoffs are, e.g.,

```javascript
if (percentGrade < 60) {
  return 'F';
} else if (percentGrade < 63) {
  return 'D-';
} else if (percentGrade < 68) {
  return 'D';
} else if {
  // etc..
} else {
  return 'A';
}